### ISSUE TITLE
Modules past compatibility maximum should display as inactive, rather than not at all

### ISSUE THREAD

**Issue by Dana-Johnson** [foundryvtt/foundryvtt#10305]
### User Experience

I had modules that were installed for a previous version and after an upgrade, the module no longer showed up in the module list.  The module developer may or may not have used the compatibility maximum correctly but even so, it would be nice to have some feedback that the module is actually installed but no longer valid for the current Foundry version.

**Comment by aaclayton** [comment]
This isn't the behavior I would expect to occur. Could you provide an example of a specific module and specific Foundry VTT version where the module disappeared from your list rather than being displayed as incompatible?

**Comment by stwlam** [comment]
It may be that the module's manifest is no longer readable in V11, rather than a maximum version being set.

**Comment by Dana-Johnson** [comment]
I have done a fresh install and can no longer replicate the problem.  I tried changing the maximum for a module back to 10 and it does show up on the list as unsupported core version now.

I am running a server on Linux and had upgraded from 9 to 10 and now to 11.  Something must have gone awry along the way.  When you said it wasn't expected behavior, I figured it must be me.  

Thank you for looking at it.  Sorry for issue...
